src/openclpow.py

```diff
@@ -96,20 +106,17 @@ def do_opencl_pow(hash, target):
             cl.enqueue_copy(queue, output, dest_buf)
         queue.finish()
         progress += globamt
-        sofar = time.time() - start
-#       logger.debug("Working for %.3fs, %.2f Mh/s", sofar, (progress / sofar) / 1000000)
     if shutdown != 0:
-        raise Exception ("Interrupted")
-    taken = time.time() - start
+        raise Exception("Interrupted")
 #   logger.debug("Took %d tries.", progress)
     return output[0][0]
 
-#initCL()
 
 if __name__ == "__main__":
```

Comment:
please add
```initCL()```
at line 116, it's an undocumented bug.

Reply:
okay .

Reply:
This hasn't been done. The main section should start with `initCL()`, like this:

```
if __name__ == "__main__":
    initCL()
    target = blahblah
    .
    .
    .
```

Reply:
Maybe better to turn this script into a test and remove shebang?

Reply:
I sometimes use it to stress test a GPU, so both a test and a standalone program make sense.